api/projects/inventory.go

```diff
@@ -123,6 +125,19 @@ func AddInventory(w http.ResponseWriter, r *http.Request) {
 	mulekick.WriteJSON(w, http.StatusCreated, inv)
 }
 
+func isValidInventoryPath(path string) bool {
+	if currentPath, err := filepath.Abs("./"); err != nil {
```

Comment:
maybe you could use os.Getwd() here instead
https://golang.org/src/os/getwd.go

Reply:
also you might need to look at using https://golang.org/pkg/os/#pkg-constants for the filepath separator to make things cross platform compatible if getwd is not suitable